raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -332,14 +332,14 @@ contract TokenNetwork is Utils {
             partner_signature
         );
 
-        require(current_withdraw > 0);
+        require(current_withdraw > 0, 'Invalid withdrawn amount.');
 
-        // Underflow check
+        // Overflow check
         require(participant_state.withdrawn_amount >= current_withdraw);
         require(participant_state.withdrawn_amount == total_withdraw);
 
         // Entire withdrawn amount must not be bigger than the entire channel deposit
-        require(participant_state.withdrawn_amount <= (total_deposit - partner_state.withdrawn_amount));
+        require(participant_state.withdrawn_amount <= (total_deposit - partner_state.withdrawn_amount), 'Invalid withdrawn amount.');
```

Comment:
This string is equal to the one used for the `0` check, I would differentiate among them, e.g. 'withdraw larger than channel deposit'